### PR TITLE
 Updating R-Host with fix for R Interrupt on linux Release 1.3

### PR DESCRIPTION
**Pull request by karthiknadig**

### CODE REVIEW

**Comment by AlexanderSher** [comment]
Verified code merge. Need someone to take a look on the text.

File: doc/rtvsd/rtvs-daemon-installation.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+4. [required for production] Configure SSL certificate: 
+By default, `rtvs-daemon` uses the `ssl-cert-snakeoil.pem` and `ssl-cert-snakeoil.pem` generated by `ssl-cert` pacakge. During installation we combine them into `ssl-cert-snakeoil.pfx`. For production purposes use the SSL certificate provided by your admin. SSL certificate can be configured by providing a PFX file and optional import password, in: `/etc/rtvs/rtvsd.config.json`
+5. [optional] Check service is running:
+`ps -A -f | grep rtvsd`
+If you don�t see a process running under the user name `rtvssvc`. Start it using the following command:
+`sudo systemctl start rtvsd`
+
+
+### On Ubuntu Server VM on Azure
+You will need Putty for windows, or other SSH client.
```
Comment (MikhailArkhipov) [inline doc/rtvsd/rtvs-daemon-installation.md]: for `Windows`

File: doc/rtvsd/rtvs-daemon-installation.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+8. [optional] Set a DNS name, you can also use the IP address.
+9. Connect to the VM using SSH client, and download the rtvs daemon tarball.
+`wget -O rtvs-daemon.tar.gz https://aka.ms/r-remote-services-linux-binary-current`
+`tar -xvzf rtvs-daemon.tar.gz`
+10. Run the preinstall script:
+`sudo ./rtvs-install`
+11. Enable and start the service:
+`sudo systemctl enable rtvsd`
+12. [required for production] Configure SSL certificate: 
+By default, `rtvs-daemon` uses the `ssl-cert-snakeoil.pem` and `ssl-cert-snakeoil.pem` generated by `ssl-cert` pacakge. During installation we combine them into `ssl-cert-snakeoil.pfx`. For production purposes use the SSL certificate provided by your admin. SSL certificate can be configured by providing a PFX file and optional import password, in: `/etc/rtvs/rtvsd.config.json`
```
Comment (MikhailArkhipov) [inline doc/rtvsd/rtvs-daemon-installation.md]: `package`

File: doc/rtvsd/rtvs-daemon-installation.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+8. [optional] Set a DNS name, you can also use the IP address.
+9. Connect to the VM using SSH client, and download the rtvs daemon tarball:
+`wget -O rtvs-daemon.tar.gz https://aka.ms/r-remote-services-linux-binary-current`
+`tar -xvzf rtvs-daemon.tar.gz`
+10. Run the preinstall script:
+`sudo ./rtvs-install`
+11. Enable and start the service:
+`sudo systemctl enable rtvsd`
+12. [required for production] Configure SSL certificate: 
+By default, `rtvs-daemon` uses the `ssl-cert-snakeoil.pem` and `ssl-cert-snakeoil.pem` generated by `ssl-cert` pacakge. During installation we combine them into `ssl-cert-snakeoil.pfx`. For production purposes use the SSL certificate provided by your admin. SSL certificate can be configured by providing a PFX file and optional import password, in: `/etc/rtvs/rtvsd.config.json`
```
Comment (MikhailArkhipov) [inline doc/rtvsd/rtvs-daemon-installation.md]: `package`

File: doc/rtvsd/rtvs-daemon-installation.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+### On Windows Subsystem for Linux (WSL)
+1. Instructions to setup WSL. 
+https://msdn.microsoft.com/en-us/commandline/wsl/install_guide
+2. Start bash on Windows and download rtvs daemon tarball:
+`wget -O rtvs-daemon.tar.gz https://aka.ms/r-remote-services-linux-binary-current`
+`tar -xvzf rtvs-daemon.tar.gz`
+3. Run the install script:
+`sudo ./rtvs-install`
+4. [required for production] Configure SSL certificate: 
+By default, `rtvs-daemon` uses the `ssl-cert-snakeoil.pem` and `ssl-cert-snakeoil.pem` generated by `ssl-cert` pacakge. During installation we combine them into `ssl-cert-snakeoil.pfx`. For production purposes use the SSL certificate provided by your admin. SSL certificate can be configured by providing a PFX file and optional import password, in: `/etc/rtvs/rtvsd.config.json`
```
Comment (MikhailArkhipov) [inline doc/rtvsd/rtvs-daemon-installation.md]: `package`

**Pull request merged**